### PR TITLE
unicode() was removed in Python 3

### PR DESCRIPTION
**Pull request by cclauss**
@hornc As discussed.

### CODE REVIEW

**Comment by hornc** [comment]
I'm still testing this, but on running this PR branch with Open Library in local dev

http://localhost:8080 returns the following:

```
  File "/usr/local/lib/python2.7/dist-packages/gunicorn/workers/base.py", line 138, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python2.7/dist-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "scripts/openlibrary-server", line 86, in load
    infogami._setup()
  File "/openlibrary/infogami/__init__.py", line 54, in _setup
    delegate._load()
  File "/openlibrary/infogami/utils/delegate.py", line 178, in _load
    __import__(plugin.module + '.code', globals(), locals(), ['plugins'])
```
And OL fails to load.
(commenting to record, and share)

File: infogami/plugins/links/code.py [hunk 1]
```diff
@@ -18,7 +18,6 @@ class backlinks (delegate.mode):
     def GET(self, site, path):
         #@@ fix later
         return []
-
-        links = tdb.Things(type=db.get_type(site, 'type/page'), parent=site, links=path)
+        # unreachable code...
```
Comment (hornc) [inline infogami/plugins/links/code.py]: This seemed very strange, but the failure to load OL seems to be from this comment line! When I try to import this file in Python 2 I get:

```
>>> import code
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "code.py", line 21
SyntaxError: Non-ASCII character '\xc2' in file code.py on line 21, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details
```

Deleting that comment fixes the problem. I couldn't see the problem using github, editors, or other tools, and copying the text seemed to ignore the strange characters, but opening this file in a hex editor shows `\xC2 \xA0` bytes between the `#` and `unreachable`, which is a Unicode no-break space in utf8.

The other way to fix would be to declare a utf8 coding in this file fot Python 2, but I'm not sure the space was deliberate?

Annoyingly this load error was _not_ picked up in Travis tests, but then again this file seems to do nothing very useful, but it was loaded by Open Library calling `infogami._setup()` from https://github.com/internetarchive/openlibrary/blob/0565e81e4e2e1ecf2fdc497e9d9bc22bf69b5685/scripts/openlibrary-server#L86

Comment (cclauss) [reply infogami/plugins/links/code.py]: Unbalanced parentheses??

Comment (cclauss) [reply infogami/plugins/links/code.py]: Oh... There is a space char that is not quite a space char.  If you retype each space char it will go away.

Comment (hornc) [reply infogami/plugins/links/code.py]: I have replaced that space character, and the site loads.

I'm only testing the changes indirectly via OL, but I was able to add a book record with Unicode title, publisher, author and other fields and search and retrieve it in local dev, so no smoke! :)

**Comment by hornc** [comment]
@mekarpeles I'd like to get this tested on OL dev at some point. Let's talk about the process on how / whether we can do that with the current server configuration.

**Comment by cclauss** [comment]
Bump on this one.

**Pull request merged**